Source/UZKFileInfo.m

```diff
@@ -13,36 +13,33 @@
  *    more version field references are available from
  *    https://www.pkware.com/documents/casestudies/APPNOTE.TXT 4.4.2.1
  */
-typedef NS_ENUM(NSUInteger, UZKZipOS) {
-    
-    UZKZipOSMSDOS = 0,
-    UZKZipOSAmiga = 1,
-    UZKZipOSOpenVMS = 2,
-    UZKZipOSUnix = 3,
-    UZKZipOSVMCMS = 4,
-    UZKZipOSAtariST = 5,
-    UZKZipOSOS2 = 6,
-    UZKZipOSMacintosh = 7,
-    UZKZipOSZSystem =8,
-    UZKZipOSZCPM = 9,
-    UZKZipOSWindowsNT = 10,
-    UZKZipOSMVS = 11,
-    UZKZipOSVSE = 12,
-    UZKZipOSAcornRisc = 13,
-    UZKZipOSVFAT = 14,
-    UZKZipOSAlternateMVS = 15,
-    UZKZipOSBEOS = 16,
-    UZKZipOSTandem = 17,
-    UZKZipOSOS400 = 18,
-    UZKZipOSDarwin = 19
+typedef NS_ENUM(NSUInteger, UZKZipOSClassicMac) {
+    UZKZipOSClassicMacMSDOS = 0,
+    UZKZipOSClassicMacAmiga = 1,
+    UZKZipOSClassicMacOpenVMS = 2,
+    UZKZipOSClassicMacUnix = 3,
+    UZKZipOSClassicMacVMCMS = 4,
+    UZKZipOSClassicMacAtariST = 5,
+    UZKZipOSClassicMacOS2 = 6,
+    UZKZipOSClassicMacMacintosh = 7,
```

Comment:
This was the only enum case that was supposed to be "ClassicMac", not the rest. This one should be `UZKZipOSClassicMac`, and `ClassicMac` should be removed from the rest.

Reply:
In order for me to understand what you mean correctly, does this modification only modify the **UZKZipOSClassicMac = 7** line?

Reply:
No, I mean that they all need to change. Lines 22-24 should be:

```
    UZKZipOSAtariST = 5,
    UZKZipOSOS2 = 6,
    UZKZipOSClassicMac = 7,
```